app/models/pipeline_run_stage.rb

```diff
@@ -216,7 +216,7 @@ def prepare_dag(attribute_dict, key_s3_params = nil)
     attribute_dict[:bucket] = SAMPLES_BUCKET_NAME
 
     # Temp flag for rolling out jbuilder templates
-    dag_ext = if AppConfigHelper.get_app_config(AppConfig::USE_JBUILDER_TEMPLATES) == "1" && step_number == 1
+    dag_ext = if AppConfigHelper.get_app_config(AppConfig::USE_JBUILDER_TEMPLATES) == "1"
```

Comment:
Would it make sense to completely remove the `.erb` files and this temp flag since you are now moving all these files to jbuilder?

Reply:
I put it there for operational safety to avoid a hotfix if we discover a problem.. :)

Reply:
but will do a follow-up to remove them maybe 1 week after it all goes to prod